astropy/coordinates/earth.py

```diff
@@ -595,14 +593,28 @@ def to_geodetic(self, ellipsoid=None):
         For the conversion to geodetic coordinates, the ERFA routine
         ``gc2gd`` is used.  See https://github.com/liberfa/erfa
         """
+        warn("Method .to_geodetic(ellipsoid) is deprecated, "
```

Comment:
Agree with the deprecation message, but in analogy with a frame or sky coord, I think there is little against still having the properties for particular representations, i.e., I think we should allow (and also refer to in the warning message),
```
el.{wgs84,...}geodetic
el.cartesian
```

Reply:
Removed the deprecation message for now, but saving this for the future.